scripts/apps/search/components/actions-menu/Item.tsx

```diff
@@ -144,13 +145,13 @@ export default class Item extends React.Component<any, any> {
             null,
             React.createElement(
                 'a',
-                {title: gettextCatalog.getString(activity.label), onClick: this.run},
+                {title: gettext(activity.label), onClick: this.run},
                 React.createElement('i', {
                     className: 'icon-' + activity.icon,
                 }),
                 React.createElement('span', {
                     style: {display: 'inline'},
-                }, gettextCatalog.getString(activity.label)),
+                }, gettext(activity.label)),
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.

Reply:
I think on activity.label we have a list of values so probably we have to translate it